operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -976,6 +977,116 @@ public Step createSteps() {
         return DomainValidationSteps.createDomainValidationSteps(getNamespace(), strategy);
       }
     }
+
+    private void runDomainPlan(DomainProcessorImpl domainProcessor, Packet packet) {
+      StepAndPacket plan = createDomainPlanSteps(packet);
+      FiberGate gate = domainProcessor.getMakeRightFiberGate(getNamespace());
+      CompletionCallback cc =
+          new CompletionCallback() {
+            @Override
+            public void onCompletion(Packet packet) {
+              // no-op
+            }
+
+            @Override
+            public void onThrowable(Packet packet, Throwable throwable) {
```

Comment:
FYI - I have changes in the same area in my PR 2609 which might conflict with these changes.

Reply:
Yes, and I'll have to deal with that.